### PR TITLE
Added lots of new plots to visualize SLA by species, plot, tag, etc.

### PR DESCRIPTION
**Pull request by lilliehaddock**
Joining: 
```{r}
Date  Tag No_Tag_Species No_Tag_DBH  Position n_Leaves Leaf_Area_cm2 Leaf_Mass_g Notes specific_leaf_area
1 6/12/2019 1758                        NA Low/Shade        5        118.53       0.439                 270.0000
2 6/12/2019 1744                        NA Low/Shade        5         68.21       0.346                 197.1387
3 6/12/2019 1810                        NA Low/Shade        5         92.85       0.359                 258.6351
4 6/12/2019 1707                        NA Low/Shade        5        146.26       0.448                 326.4732
5 6/12/2019 1814                        NA Low/Shade        5         85.77       0.305                 281.2131
6 6/12/2019 1714                        NA Low/Shade        5         45.80       0.356                 128.6517
  Plot Species
1 Salt    FAGR
2 Salt    FAGR
3 Salt    FAGR
4 Salt    FAGR
5 Salt    FAGR
6 Salt    FAGR
```
image

image

### CODE REVIEW

**Comment by bpbond** [comment]
Nice. What about plotting DBH (x) versus SLA (y), coloring by species? Seems like that would be a useful plot.

File: SLA.R [hunk 1]
```diff
@@ -28,4 +32,69 @@ split_by_date <- ggplot(data = sla, aes(n_Leaves, Leaf_Area_cm2, color = Date))
...
+## SLA by Plot
+sla_plot1 <- sla_joined %>% 
+  ggplot(aes(Species, specific_leaf_area, color = Position)) +
+  geom_point() +
+  geom_jitter() +
+  theme_bw() +
+  facet_wrap(~Plot) +
+  theme(axis.text.x = element_text(angle = 90)) +
+  labs(title = "Specific Leaf Area by Plot", y = "Specific Leaf Area", 
+       subtitle = "Lillie Haddock 6/21/19")
```
Comment (haileymoore) [inline SLA.R]: Might want to consider removing the subtitle, if you use the script again for new data it'll be annoying to have to go in and change the date every time

File: SLA.R [hunk 2]
```diff
@@ -28,4 +32,69 @@ split_by_date <- ggplot(data = sla, aes(n_Leaves, Leaf_Area_cm2, color = Date))
...
+  summarize(mn = mean(DBH), sd = sd(DBH))
+
+# Create new dataset with plot, species, and tag columns
+# JOIN!!! sla and inventory_small
+inventory_small <- inventory %>% 
+  select(Plot, Species, Tag) 
+sla_joined <- left_join(sla, inventory_small)
+
+## SLA by Plot
+sla_plot1 <- sla_joined %>%
```
Comment (haileymoore) [inline SLA.R]: Make the plot names more specific rather than numbering

**Review by haileymoore — CHANGES_REQUESTED** [review]

**Review by marideeweber — APPROVED** [review]

File: SLA.R (hunk 1 above)
Comment (bpbond) [reply SLA.R]: Agreed. Better though: `subtitle = paste("Lillie Haddock", date())`

**Pull request merged**